tests/unit/include/SugarEmailAddress/SugarEmailAddressTest.php

```diff
@@ -982,7 +984,7 @@ public function testGetEmailGUID()
 
         // test
         $result = $this->ea->getEmailGUID('test@email.com');
-        self::assertEquals($result, 'test_email_1');
+        self::assertEquals('test_email_1', $result);
```

Comment:
why?

Reply:
The first param is the "expected" value, second param is the "actual" value. These are the wrong way around, which is rather confusing.

Reply:
ok